stock_manager/controllers/add.py

```diff
@@ -0,0 +1,121 @@
+from typing import override, TYPE_CHECKING
+
+from PyQt6.QtWidgets import QLineEdit, QMessageBox, QSpinBox, QTextEdit
+
+import stock_manager
+from .abstract import AbstractController
+
+if TYPE_CHECKING:
+	from stock_manager import App
+
+
+class Add(AbstractController):
+	def __init__(self, app: 'App'):
+		super().__init__('add', app)
+		
+		self._total = self._excess = 0
+		self._spinners: list[QSpinBox] = [
+			self.b750_spinner,
+			self.b757_spinner,
+			self.min_750_spinner,
+			self.min_757_spinner
+		]
+		self._text_fields: list[QLineEdit | QTextEdit] = [
+			self.part_num,
+			self.manufacturer,
+			self.desc
+		]
+		self.PAGE_INDEX = 3
```

Comment:
Beware magic numbers here.  If we want to change this index, we have to search through the rest of the page files and manually verify that the others are still correct.

In the future I think it's worth setting up a more automatic way of assigning the page indices